src/components/User/KycRequests/KycRequest.List.vue

```diff
@@ -217,6 +214,20 @@ export default {
       this.list = data
     },
 
+    async setFilters () {
+      if (this.specificUserAddress) {
+        const emailRegExp = /^[^\s@]+@[^\s@]+\.[^\s@]+$/
+        const idLength = 56
```

Comment:
use the same "accountId or email" check that you got for `AssetRequestList.vue `